outputs.tf

```diff
@@ -53,12 +53,12 @@ output "ecs_security_group" {
 
 output "ecs_cluster_id" {
   description = "ECS cluster id"
-  value       = module.ecs.ecs_cluster_id
+  value       = local.ecs_cluster_id
 }
 
 output "ecs_cluster_arn" {
   description = "ECS cluster ARN"
-  value       = module.ecs.ecs_cluster_arn
+  value       = local.ecs_cluster_arn
```

Comment:
according to the docs, both ID and ARN are the same (ARN). @antonbabenko what if we just set these both to `value = local.ecs_cluster_id`? or should we leave this as `module.ecs.cluster_arn` knowing it will be empty when you don't create a cluster with this module?

either way, we should drop the `ecs_cluster_arn` variable, its not providing value here since we are asking for the ID. lets remove that variable and the associated local variable, thanks!

Reply:
Happy to remove it, I was just maintaining existing compatibility.

Reply:
+1 @bryantbiggs as long as it works for both cases. `local.ecs_cluster_id` sounds valid to me.